### DISCUSSION TITLE
Ad-network mediation

### DISCUSSION THREAD

**Discussion by Chadori** [admob-plus/admob-plus#440] [Fund Features]
Hi there,
I am willing to fund ad-network mediation support, or at least allow me to make pull-requests to add them.
If funding is allowed, please allow me to officially make pull-requests to make changes as well.

Can you please provide me here your email address to make a deal? Thank you.

**Comment by EMI-INDO** [comment]
We have collected Ad-network mediation funds.

Check here https://github.com/admob-plus/admob-plus/discussions/314

Now only 35% collected

Each new feature costs around $1000
